src/reducers/entities/posts.js

```diff
@@ -28,12 +28,24 @@ function handleReceivedPost(posts = {}, postsInChannel = {}, action) {
         ];
     }
 
-    return {posts: nextPosts, postsInChannel: nextPostsForChannel};
+    let nextPostsForThread = postsInThread;
```

Comment:
Can you fix this naming? You're mixing up ForThread vs InThread in a few of these functions. It looks like you want postsInThread to be the field in the state and postsForThread to be `postsInThread[rootId]`

Reply:
It is a bit mixed up but I directly copied the naming from the `postsInChannel`. I think it's better to match that don't you think?

Reply:
Actually I'll just fix them both